apps/web/app/(use-page-wrapper)/[user]/[type]/page.tsx

```diff
@@ -36,7 +36,8 @@ export const generateMetadata = async ({ params, searchParams }: PageProps) => {
     (t) => `${rescheduleUid && !!booking ? t("reschedule") : ""} ${title} | ${profileName}`,
     (t) => `${rescheduleUid ? t("reschedule") : ""} ${title}`,
     isBrandingHidden,
-    getOrgFullOrigin(eventData?.entity.orgSlug ?? null)
+    getOrgFullOrigin(eventData?.entity.orgSlug ?? null),
+    `/${params.user}/${params.type}`
```

Comment:
<img width="551" alt="Screenshot 2025-02-14 at 8 45 15 AM" src="https://github.com/user-attachments/assets/34aa5854-eb96-4318-ac71-60b2532fd2ca" />

`og:url` is being created correctly